include/bitcoin/bitcoin/settings.hpp

```diff
@@ -40,6 +40,11 @@ class BC_API settings
     //--------------------------------------------------------------------------
 
     uint64_t bitcoin_to_satoshi(uint64_t bitcoin_units=1) const;
+    void initial_block_subsidy_bitcoin(uint64_t initial_block_subsidy_bitcoin);
+    uint64_t initial_block_subsidy_bitcoin() const;
+    void subsidy_interval(uint64_t subsidy_interval);
+    uint64_t subsidy_interval() const;
+    uint64_t max_money() const;
```

Comment:
It seems inevitable when creating a class with fields that eventually proper behavior will drive it into properties.